src/modules/likes.js

```diff
@@ -0,0 +1,49 @@
+export const likesCounter = (data, id) => {
+  const likeArryLength = data.length;
+  let likesNumber;
+  if (likeArryLength === 0) {
+    likesNumber = 0;
+    return likesNumber;
+  }
+  likesNumber = data[id].likes;
+  return likesNumber;
+};
+
+export const fetchLikes = async (url) => {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    // likesCounter(data);
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented code as it makes your repository clean and readable.